projects/storefrontlib/src/cms-components/navigation/footer-navigation/scroll-to-top/scroll-to-top.component.ts

```diff
@@ -0,0 +1,49 @@
+import { Component, HostListener, OnInit } from '@angular/core';
+
+const SCROLL_TO_TOP_TRUE = 100;
```

Comment:
These consts should be a part of the component, so they can be changed/extended in the future, by the customer

Reply:
You mentioned, components. But the constants are part of "scroll-to-top.component.ts" already.
Which component should it be part of?